ldms/src/ldmsd/ldmsd_failover.c

```diff
@@ -2079,7 +2086,7 @@ int failover_cfgprdcr_handler(ldmsd_req_ctxt_t req)
 	}
 
 	p = ldmsd_prdcr_new_with_auth(name, xprt, host, atoi(port), ptype,
-			atoi(interval), auth, _uid, _gid, _perm);
+			atoi(interval), auth, _uid, _gid, _perm, atoi(cache_ip));
```

Comment:
what happened if `cache_ip` is `NULL`?

Reply:
more significantly, afaict ip may be 64 bits, so atoi is not necessarily a valid  conversion.

Reply:
I added the `cache_ip` value to the producer information message sent by the buddy ldmsd, so it isn't NULL, the same as the other attributes.